src/renderer/viz/expressions/placement.js

```diff
@@ -7,14 +7,6 @@ import { checkLooseType, checkType, implicitCast } from './utils';
  * - `symbolPlacement: placement(0,0)` means to align the center of the image with the point center.
  * - `symbolPlacement: placement(-1,-1)` means to align the top right corner of the image with the point center.
  *
- *           |1
- *           |
- *           |
- * -1 -------+------- 1
```

Comment:
IMO, this is useful

Reply:
This ASCII picture generates very bad results:

![selection_257](https://user-images.githubusercontent.com/2227572/44262177-61e44780-a21a-11e8-90bd-fab425f19cac.png)

Reply:
Can't we fix that instead of removing it?

Reply:
+1 to fix it 🙏

Reply:
:+1: I move it inside a code block